articles/automation/automation-hybrid-runbook-worker.md

```diff
@@ -136,10 +136,13 @@ In addition to the standard addresses and ports that the Hybrid Runbook Worker r
 |*.oms.opinsights.azure.com     | *.oms.opinsights.azure.us        |
 |*.blob.core.windows.net|*.blob.core.usgovcloudapi.net|
 
+### Using an OMS gateway
+When using an OMS gateway to connect machines to OMS, the OMS gateway needs to be configured for hybrid workers to function correctly. [Find the configuration steps here](https://docs.microsoft.com/en-us/azure/log-analytics/log-analytics-oms-gateway#configure-for-automation-hybrid-workers)
```

Comment:
Please delete this section as it is redundant. In the beginning of the main section we already talk about the OMS gateway.